Dont set capabilities on kwin_wayland executable at build time

This is handled by security.wrappers on NixOS

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 48cbcdbfe..93b410ed8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -674,15 +674,6 @@ if (HAVE_LIBCAP)
 endif()

 install(TARGETS kwin_wayland         ${INSTALL_TARGETS_DEFAULT_ARGS} )
-if (HAVE_LIBCAP)
-    install(
-    CODE "execute_process(
-            COMMAND
-                ${SETCAP_EXECUTABLE}
-                CAP_SYS_NICE=+ep
-                \$ENV{DESTDIR}${CMAKE_INSTALL_FULL_BINDIR}/kwin_wayland)"
-    )
-endif()

 add_subdirectory(platformsupport)
 add_subdirectory(plugins)
